stories/legacy.stories.js

```diff
@@ -128,28 +128,9 @@ export const WMSLegacyGetMapFromUrlDatesNotTimes = () => {
 };
 
 async function setAuthTokenWithOAuthCredentials() {
-  let authToken;
```

Comment:
To avoid authenticating every time a new image is fetched (we must set a good example ;) ), I would suggest adding a check here:
```
  if (isAuthTokenSet()) {
    return;
  }
```

EDIT: same in `index.stories.js`.

Reply:
Because of the nature of the `isAuthTokenSet()` function, it returns `true` when the `authToken` is set to `undefined`.  Should we change that so it returns `false` if the `authToken` is `undefined`?

```javascript
export function isAuthTokenSet(): boolean {
  return authToken !== null;
}

Reply:
Yes please! :)

I guess in the future it should also check the auth token expiry time, not top priority yet though (our existing apps handle auth token updating already).

Reply:
I changed my mind a little bit - I changed the `setAuthToken()` function so it throws an Error if the parameter is `undefined`. Now we fully comply with the type of `authToken` (either `string` or `null`).